core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2278,6 +2278,37 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_BLOCK_MASTER_CLIENT_POOL_SIZE_MIN =
+      new Builder(Name.USER_BLOCK_MASTER_CLIENT_POOL_SIZE_MIN)
+          .setDefaultValue(0)
+          .setDescription("The minimum number of block master clients cached in the block master "
+              + "client pool. For long running processes, this should be set to zero.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.CLIENT)
+          .build();
+  public static final PropertyKey USER_BLOCK_MASTER_CLIENT_POOL_SIZE_MAX =
+      new Builder(Name.USER_BLOCK_MASTER_CLIENT_POOL_SIZE_MAX)
+          .setDescription("The maximum umber of block master clients cached in the block master "
+              + "client pool.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.CLIENT)
+          .setAlias(new String[] {Name.USER_BLOCK_MASTER_CLIENT_THREADS})
```

Comment:
Are these supposed to be the same number? Does this mean `USER_BLOCK_MASTER_CLIENT_THREADS` should be deprecated?

Reply:
yes